design-documents/graph-ql/coverage/gift-registry.graphqls

```diff
@@ -0,0 +1,278 @@
+type Customer {
+    gitft_registry_list: [GiftRegistry]
+    gitft_registry(id: ID!): GiftRegistry
+}
+
+type Query {
+    giftRegistryTypes: [GiftRegistryType] @doc(description: "Get a list of available gift registry types")
+    giftRegistrySearch(
```

Comment:
That's our current agreement, we have already discussed recently whether it makes sense to change, but seems like because backward compatibility better to keep this style. Feel free to create a separate proposal/discussion on this topic.

Reply:
Sounds good - if it's already a thing no need to discuss in this PR 👍